runtime/include/chplmath.h

```diff
@@ -34,6 +34,8 @@ static inline int chpl_macro_double_isfinite(double x) { return isfinite(x); }
 static inline int chpl_macro_float_isfinite(float x) { return isfinite(x); }
 static inline int chpl_macro_double_isnan(double x) { return isnan(x); }
 static inline int chpl_macro_float_isnan(float x) { return isnan(x); }
+static inline bool chpl_macro_double_signbit(double x) { return signbit(x)!=0?true:false; }
+static inline bool chpl_macro_float_signbit(float x) { return signbit(x)!=0?true:false; }
```

Comment:
I don't think we want to the return type to be `bool` (for portability reasons.) It could be `chpl_bool`, but then you'd have to include `chpltypes.h`, which we probably want to avoid. 

It might be better to follow `chpl_macro_float_isnan`. Have the return type be an int and just return `signbit(x)` and, and do the cast to bool in the module code.

Reply:
Did it, please check once.